### PR TITLE
Remove deprecated and unused Objective-C services

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
Removed `LaunchServicesWrapper`, which uses a deprecated API to set `.ipa`s to open with PlayCover by default without alerting the user.

Removed `LoginKitWrapper`, which uses a deprecated API to manage login items, and is unused.

Fixed a deprecation warning.

### CODE REVIEW

**Comment by IsaacMarovitz** [comment]
Rebasing and resubmitting bare with me 💀

File: PlayCover/View/PlayCoverApp.swift [hunk 1]
```diff
@@ -31,7 +31,6 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 		UserDefaults.standard.register(
 			defaults: ["NSApplicationCrashOnExceptions" : true]
 		)
-        LaunchServicesWrapper.setMyselfAsDefaultApplicationForFileExtension("ipa")
```
Comment (zanderp25) [inline PlayCover/View/PlayCoverApp.swift:34]: Doesn't it need this to set itself as a default for ipa files, or is it done somewhere else?

Comment (IsaacMarovitz) [reply PlayCover/View/PlayCoverApp.swift:34]: It was Candy's opinion that setting PlayCover to open `.ipa`s by default without alerting the user isn't great practice. Elsewhere in the `Info.plist` it specifies that PlayCover can open `.ipa` files so it should still be available as an option, but just wont be set as default.

Comment (zanderp25) [reply PlayCover/View/PlayCoverApp.swift:34]: Ah, ok. Makes sense.

**Comment by IsaacMarovitz** [comment]
Removed animation deprecation fix. Now part of #96.

**Review by Depal1 — APPROVED** [review]

**Pull request merged**